tests/ExamplePluginWithLib/Upstream/README.md

```diff
@@ -0,0 +1,23 @@
+# Plugin sharing library - Downstream
+This example illustrates how to ship a library with a Radium plugin, and make the library available
+to other plugins.
+
+## Configure and compile
+```bash
+mkdir build
+ce build
+cmake -DRadium_DIR=/pathToInstalledRadium/lib/cmake/Radium/  -DQt5_DIR=/pathToInstalledQt5/lib/cmake/Qt5 ..`
```

Comment:
```suggestion
cmake -DRadium_DIR=/pathToInstalledRadium/lib/cmake/Radium/..`
```

Reply:
As the plugin search for the package Qt, this must be given to cmake if Qt5 is not installed on "standard" location. This is mostly (and quite always) the case on Windows or MacOs where the user might install Qt5 in user space, or system-wide, in whatever directory `/pathToInstalledQt5` he wants.